iotix/patterns/startup.php

```diff
@@ -40,39 +40,39 @@
 <div class="wp-block-columns alignwide"><!-- wp:column {"width":"50%","style":{"border":{"width":"0px","style":"none"}}} -->
 <div class="wp-block-column" style="border-style:none;border-width:0px;flex-basis:50%"><!-- wp:group {"style":{"border":{"radius":"30px"},"spacing":{"padding":{"top":"var:preset|spacing|70","right":"var:preset|spacing|70","bottom":"var:preset|spacing|70","left":"var:preset|spacing|70"}}},"backgroundColor":"background","layout":{"type":"constrained"}} -->
 <div class="wp-block-group has-background-background-color has-background" style="border-radius:30px;padding-top:var(--wp--preset--spacing--70);padding-right:var(--wp--preset--spacing--70);padding-bottom:var(--wp--preset--spacing--70);padding-left:var(--wp--preset--spacing--70)"><!-- wp:heading -->
-<h2 class="wp-block-heading"><strong><?php echo esc_html__( 'No Code Necessary', 'iotix' ); ?></strong></h2>
+<h2 class="wp-block-heading"><strong><?php esc_html_e( 'No Code Necessary', 'iotix' ); ?></strong></h2>
 <!-- /wp:heading -->
 
 <!-- wp:paragraph {"fontSize":"medium"} -->
-<p class="has-medium-font-size"><?php echo esc_html__( 'Not a programmer? Use our intuitive, designer-first UI.', 'iotix' ); ?></p>
+<p class="has-medium-font-size"><?php esc_html_e( 'Not a programmer? Use our intuitive, designer-first UI.', 'iotix' ); ?></p>
 <!-- /wp:paragraph -->
 
 <!-- wp:image {"sizeSlug":"large","linkDestination":"none","className":"is-style-default"} -->
 <figure class="wp-block-image size-large is-style-default"><img src="<?php echo esc_url( get_stylesheet_directory_uri() ); ?>/assets/images/f24d9-column_1_image.webp" alt="" /></figure>
 <!-- /wp:image -->
 
 <!-- wp:paragraph {"fontSize":"medium"} -->
-<p class="has-medium-font-size"><a href="https://iotixdemo.wordpress.com/"><?php echo esc_html__( 'Try a demo →', 'iotix' ); ?></a></p>
+<p class="has-medium-font-size"><a href="https://iotixdemo.wordpress.com/"><?php esc_html_e( 'Try a demo →', 'iotix' ); ?></a></p>
```

Comment:
Should this be using a placeholder url rather than https://iotixdemo.wordpress.com/?